### DISCUSSION TITLE
控制台504错误

### DISCUSSION THREAD

**Discussion by aloeJun** [spencerwooo/onedrive-vercel-index#292] [Q&A] [answered]
https://one-drive-vercel-30hw4pis9-ialoe.vercel.app/ 部署网址，
https://github.com/ialoe/onedrive-vercel-index 库地址，
麻烦大佬有空看一下问题出在哪里
![image](https://user-images.githubusercontent.com/69854983/150279471-15d513fc-7a09-4bf0-a474-6075f69e78f2.png)

**Comment by spencerwooo** [comment, marked as answer]
你改了 client id 和 client secret，那么你的 client secret 一样需要加密，要不然你 OAuth 那一步肯定做的不对

https://docs.spencerwoo.com/using-your-own-client-id-and-secret

**Reply by aloeJun** [reply]
我又改了一下，还是没有成功

**Reply by spencerwooo** [reply]
https://docs.spencerwoo.com/error-on-deployment#6b3bd3d751fb4f8ca6bade72d0eac512

**Reply by spencerwooo** [reply]
Error 504 一般就这些问题，看看能不能解决？

**Reply by aloeJun** [reply]
我没有用redis，这个是必须的么

**Reply by aloeJun** [reply]
我开通了365开发者订阅 会有影响么
![image](https://user-images.githubusercontent.com/69854983/150331099-cf939ff6-a7b7-482f-8018-7301292cad1b.png)

**Reply by spencerwooo** [reply]
Redis 在新版本是必须的

**Reply by aloeJun** [reply]
感谢帮助，在完成redis的配置后完成了搭建

**Reply by spencerwooo** [reply]
话说，我觉得文档写的挺清楚的每一步，为什么你会觉得 Redis 可以不用配置呢？

**Reply by aloeJun** [reply]
我是在百度搜到的教程，不是直接看的文档，在百度又找到你这个库的

**Reply by spencerwooo** [reply]
呃，好吧，百度……
